samples/snippets/src/main/java/com/example/speech/SpeechModelAdaptationBeta.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.speech;
+
+// [START speech_transcribe_with_model_adaptation_gcs]
```

Comment:
@alixhami Alix, I wonder if we should pre/post-fix samples written for beta clients `beta_` so that it is easier to track them when we deprecate the endpoints.

Reply:
Caught up with Alix offline - please post-fix this sample with `_beta` so that we can properly keep track of it. Make sure to also update the region tag in DevRel services.

Reply:
I postfixed it with ```_beta```, if that's okay.

Reply:
It's not updated here - please update the region tag here.